client/dashboard/domains/domains-contact-details/style.scss

```diff
@@ -0,0 +1,7 @@
+.domains-contact-details__list {
```

Comment:
This is my opinion, but I think we can probably just inline these styles. I Recognize that's a bit less dry, but I think it's better to have less stylesheets in our project to help [communicate the idea](https://github.com/Automattic/wp-calypso/blob/trunk/client/dashboard/docs/ui-components.md#:~:text=Minimal%20CSS%3A%20Avoid%20custom%20CSS%20as%20much%20as%20possible%2C%20preferring%20component%20composition) that we don't want custom styles. The more stylesheets we have, the more likely developers (and coding agents) are to add to it without questioning whether they are needed.